src/app/shared/report-filters/report-filters.component.ts

```diff
@@ -210,7 +210,14 @@ export class ReportFiltersComponent implements OnInit, ControlValueAccessor, Aft
   }
 
   public ngOnInit() {
+    if (this.reportType === 'cervical-cancer-screening-numbers') {
+      this.genderOptions = this.genderOptions.filter(option => {
+        return option.value !== 'M' && option.label !== 'M';
```

Comment:
This is ambiguous. What if option.value == m or option.label == 'm' ?

Reply:
Is this the only case you are checking for where **option.value not equal to M** and  **option.label not equal to M**. is there any possibility either option.label or option.value can be null

Reply:
No. The genderOptions array will always be initialized to:

```
public genderOptions: Array<any> = [
    {
      value: 'F',
      label: 'F'
    },
    {
      value: 'M',
      label: 'M'
    }
  ];
```
In this case, I'd want to filter out the nested object with the male options when the current report is the cervical cancer screening report.

Reply:
👍🏽